app/services/standardize_address_service.rb

```diff
@@ -0,0 +1,81 @@
+require 'net/http'
+require 'uri'
+
+# PLEASE do NOT use this service to standardize addresses in bulk (do NOT use in backfills)
+# as that violates the terms of agreement with USPS and will result in access being revoked.
+class StandardizeAddressService
+  def initialize(intake)
+    @street_address = [intake.street_address, intake.street_address2].join(" ")
```

Comment:
i might prefix these with @_street_address etc to make it clear that these are the INPUTS (since the default behavior of many rails classes is to create attr_accessors that allow you to get the value of an instance variable as a method of the same name, and here you'll be subverting that expectation since the instance variables are the input and the methods are the output)